CMakeLists.txt

```diff
@@ -5,6 +5,14 @@ set(CMAKE_CXX_STANDARD 20)
 project(d2tm)
 
 #SET(OpenGL_GL_PREFERENCE "GLVND")
+SET(D2TM_VERSION "0.7.0")
```

Comment:
can we get this from an `environment variable` ? And if it does not exist fall back to a default value? That would make it possible to start producing binaries from GH Actions with tags for example.

Reply:
I don't know. I'm just showing you this possibility that saves you from digging through the code, and at the same time an elegant way to communicate from CMake to the program.

Reply:
Good new: after researches, we can use `environment variable`
CMake allow this: `$ENV{VARIABLE_NAME}`
You can write this `SET(D2TM_VERSION "$ENV{D2TM_VERSION_ENV}")`